### PR TITLE
Toolbar Overview: Change demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Toolbar/Overview/description.md [hunk 1]
```diff
@@ -1 +1,35 @@
-The Toolbar is&nbsp;a&nbsp;UI component containing items that usually manage screen content. Those items can be&nbsp;plain text or&nbsp;widgets.
\ No newline at end of file
+The Toolbar contains items that manage the page content: other components and screen view. In this demo, the Toolbar manages the [List](/Documentation/ApiReference/UI_Components/dxList/).
```
Comment (albertov05) [inline JSDemos/Demos/Toolbar/Overview/description.md]: `other components`

Can we make this more specific and list a few examples ("components such as _____")?

What do you mean by "screen view"? Is this the name of a component as well?

File: JSDemos/Demos/Toolbar/Overview/description.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+
+## Configure Toolbar Items
+
+You can store Toolbar items in the [items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array or specify the [dataSource](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#dataSource). A Toolbar item may be plain text or a UI component. You should specify the [text](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#text) or the [widget](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#widget) property depending on the item. If the item is a UI component, declare its [options](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#options).
```
Comment (albertov05) [inline JSDemos/Demos/Toolbar/Overview/description.md]: `You can store Toolbar items in the [items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array or specify the [dataSource](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#dataSource).`

I want to make sure my understanding of this sentence is correct.
Is "store" the correct verb here, or do you mean that the toolbar items are _displayed_ from an array?
For example, do you mean the following?

`You can display Toolbar items from an [items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array or a [dataSource](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#dataSource).`

If not, please clarify.

File: JSDemos/Demos/Toolbar/Overview/description.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+
+## Specify Item Location
+
+You can set the **dataSource** with **location** fields or specify the [location](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#location) property for each item. The **location** value can be on of the following:
```
Comment (albertov05) [inline JSDemos/Demos/Toolbar/Overview/description.md]:
```suggestion
You can set the **dataSource** with **location** fields or specify the [location](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#location) property for each item. The **location** value can be one of the following:
```

File: JSDemos/Demos/Toolbar/Overview/description.md [hunk 4]
```diff
@@ -1 +1,35 @@
...
+Additionally, the Toolbar can render its items in the overflow menu. Specify the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property for each item with one of the following values:
+
+- *"always"*  
+ Always places the item in the overflow menu.
+
+- *"never"*  
+ Places the item outside of the overflow menu.
+
+- *"auto"*  
+ Places the item outside of the overflow menu. If the width of the Toolbar is not enough to store all the items, the Toolbar renders this item in the overflow menu.
```
Comment (albertov05) [inline JSDemos/Demos/Toolbar/Overview/description.md]: `If the width of the Toolbar is not enough to store all the items, the Toolbar renders this item in the overflow menu.`

I'd like to update "not enough". Can we change this sentence to one of the following?

`If the width of the Toolbar cannot store all items, the Toolbar renders this item in the overflow menu.`

or

`If all items cannot fit within the width of the Toolbar, it renders this item in the overflow menu.`

File: JSDemos/Demos/Toolbar/Overview/description.md (hunk 2 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Toolbar/Overview/description.md]: Yes, "store" is not right, thank you for your suggestion, I will update it in the following commit.

File: JSDemos/Demos/Toolbar/Overview/description.md (hunk 4 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Toolbar/Overview/description.md]: I'm going to change it to the last one.

File: JSDemos/Demos/Toolbar/Overview/description.md (hunk 1 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Toolbar/Overview/description.md]: I decided to rerwite this sentence

File: JSDemos/Demos/Toolbar/Overview/description.md (hunk 2 above)
Comment (albertov05) [reply JSDemos/Demos/Toolbar/Overview/description.md]: Thank you.

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**